pywren/jobrunner.py

```diff
@@ -69,10 +68,17 @@ def write_stat(stat, val):
 
     for m_filename, m_data in loaded_func_all['module_data'].items():
         m_path = os.path.dirname(m_filename)
-
-        if len(m_path) > 0 and m_path[0] == "/":
-            m_path = m_path[1:]
+        if sys.platform == 'win32':
```

Comment:
Can we put this functionality into a util function ?

Reply:
Did you mean line 75?

Reply:
I meant this whole block actually